developer-docs/upgrade/index.md

```diff
@@ -4,147 +4,126 @@ page_title: Build and Deploy
+| # Neo4j VM changes - To be done on Sunbird and VDN<br># Run same steps in each node of neo4j VM<br><br>wget https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_linux-x64_bin.tar.gz<br>tar -xf openjdk-11+28_linux-x64_bin.tar.gz<br>sudo mv jdk-11 /opt<br>rm -rf openjdk-11+28_linux-x64_bin.tar.gz<br>/opt/jdk-11/bin/java -version<br># The above command should show java version as 11<br><br><br><br>sudo systemctl stop logstash.service<br>sudo su - learning<br>wget https://artifacts.elastic.co/downloads/logstash/logstash-6.8.21.tar.gz<br>tar -xf logstash-6.8.21.tar.gz<br>rm -rf logstash-6.8.21.tar.gz<br>cd /home/learning/logstash-6.3.1/<br>cp .sincedb_learning_graph_event_neo4j ../logstash-6.8.21/<br>cd ..<br>mv logstash-6.3.1/ logstash-6.3.1_bak<br>ls -larth logstash-6.8.21<br># The above command should show all the .sincedb files copied<br>exit<br><br><br># Added / Update these under [Service] section. Comment out the exisiting Environment and ExecStart and use the below ones<br>sudo vi /etc/systemd/system/logstash.service<br>Environment="JAVA_HOME=/opt/jdk-11"<br>ExecStart=/home/learning/logstash-6.8.21/bin/logstash -f /home/learning/logstash-6.8.21/logstash-lp.conf -w 1 --verbose<br>sudo systemctl daemon-reload<br><br># Redeploy neo4j service with new tag<br># Verify logstash version is 6.8.21<br>ps -ef \| grep logstash |  |  |
+
 
 ##### Note: The below jobs are applicable only if you are running Vidyadaan infrastructure
 
 ### Variables for Vidyadaan
```

Comment:
@rjshrjndrn please make the required changes as per the comment above. Please make the modifications in sections below as well.

Reply:
![image](https://user-images.githubusercontent.com/2563385/146929695-a284726f-d9ae-41d5-930b-1558fb053ab9.png)
Vidyadaan is another platform to upload content. The only adopter as of now using that is Diksha.